### PR TITLE
Few small fixes

### PR DESCRIPTION
**Pull request by anisimkov**

### CODE REVIEW

**Comment by briot** [comment]
phisical => physical

File: config/ssl/aws-net-ssl__gnutls.adb [hunk 1]
```diff
@@ -1808,7 +1813,10 @@ package body AWS.Net.SSL is
 
       Session_Transport (Socket);
 
-      if Host /= "" then
+      if Host /= "" and then not (To_Set (Host) <= IP_Address_Characters) then
+         --  GNUTLS does not allow to set phisical address text representation
```
Comment (TurboGit) [inline config/ssl/aws-net-ssl__gnutls.adb]: phisical -> physical

File: regtests/0317_hide_id/test.out [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+
+@@@ SENT ( 189/ 189 buffer usage) @@@
 
 Connection: Close
 Host:
 Accept: text/html, */*
 Accept-Encoding: deflate, gzip
 Accept-Language: fr, ru, us
-User-Agent: AWS (Ada Web Server) v19.0
+User-Agent: AWS (Ada Web Server) v20.0
```
Comment (TurboGit) [inline regtests/0317_hide_id/test.out]: this should have been filtered out, my mistake. will fix.

**Comment by anisimkov** [comment]
I masked the AWS version in 0317 test

**Pull request merged**